demos/css/CSSParserDemo.htm

```diff
@@ -2,7 +2,9 @@
 <html>
 <head>
 <title>CSS Parser Demo</title>
-<script type="text/javascript" src="../../build/parserlib.js"></script>
+<!--<script type="text/javascript" src="../../dist/parserlib.js"></script>-->
```

Comment:
I should have more of an explanation there.  I was trying to demonstrate (and test) that you can use either the all-in-one file or the two split up files.  I wonder how many people actually care about that, though?  It seems like the idea was that this was going to be a general purpose parser library... but then it turned out not to be so.